xtask/src/main.rs

```diff
@@ -60,7 +62,7 @@ fn try_main() -> Result<()> {
 
 /// The metadata of a cargo workspace.
 #[derive(Clone, Debug, Deserialize)]
-pub struct Metadata {
+pub(crate) struct Metadata {
```

Comment:
Since this is defined at the crate root, removing the `pub(crate)` entirely should still work.